tensorboard/components/tf_data_selector/tf-data-selector.ts

```diff
@@ -69,6 +82,43 @@ Polymer({
         newComparingExpIds);
   },
 
+  _canCompareExperiments(): boolean {
+    return Boolean(this._comparingExps.length);
+  },
+
+  /**
+   * Prunes away an experiment that has been removed from `_comparingExps` from
+   * the _selectionMap.
+   */
+  _pruneSelection() {
+    if (!this._canCompareExperiments()) return;
```

Comment:
Should we perhaps clear out _selectionMap in this case?

Reply:
Good call! Done!